mne/viz/_brain/_brain.py

```diff
@@ -727,6 +727,8 @@ def ensure_minimum_sizes(self):
             mpl_h = int(round((sz[1] * self.interactor_fraction) /
                               (1 - self.interactor_fraction)))
             self.mpl_canvas.canvas.setMinimumSize(sz[0], mpl_h)
+            paint_func = self.plotter.paintEvent
+            self.plotter.paintEvent = lambda ev: None
```

Comment:
This is close but suffers in terms of DRY and does not put the restoration inside a `try/except/finally`. I would do it like:
```
from contextlib import contextmanager

def _noop(*args, **kwargs):
    return

def _qt_disable_paint(widget):
    paintEvent = widget.paintEvent
    widget.paintEvent = _noop
    try:
        yield
    finally:
        widget.paintEvent = paintEvent
```

then
```
with _qt_disable_paint(self.plotter):
    ...
```
putting just whatever lines are likely to cause problems inside this context manager